bin/cli.js

```diff
@@ -3,36 +3,150 @@
 
 const fs = require('fs');
 const path = require('path');
-
+const yargs = require('yargs');
```

Comment:
Can we implement this without yargs?

Reply:
We can use [Commander](https://www.npmjs.com/package/commander) if you want. It provides similar functionality but doesn't have dependencies. And we also use [yargs](https://www.npmjs.com/package/yargs) in [jstp cli](https://github.com/metarhia/jstp/blob/master/bin/cli.js). I don't mind either option. @nechaido @belochub @lundibundi WDYT?